internal/support/notifications/application/channel/sender.go

```diff
@@ -39,9 +43,13 @@ func (sender *RESTSender) Send(notification models.Notification, address models.
 	if !ok {
 		return "", errors.NewCommonEdgeX(errors.KindContractInvalid, "fail to cast Address to RESTAddress", nil)
 	}
-	// NOTE: Not currently passing an AuthenticationInjector here;
-	// no current notifications are calling EdgeX services
-	return utils.SendRequestWithRESTAddress(lc, notification.Content, notification.ContentType, restAddress, nil)
+	var injector interfaces.AuthenticationInjector
+	if restAddress.InjectEdgeXAuth {
+		injector = secret.NewJWTSecretProvider(sender.secretProvider)
+	} else {
+		injector = nil
+	}
```

Comment:
`else` not needed since `injector` default value from `var` line is `nil`
```suggestion
	} 
```